.azure/gpu-tests.yml

```diff
@@ -12,15 +12,26 @@ trigger:
       - "master"
       - "release/*"
       - "refs/tags/*"
+  paths:
+    include:
+      - ".azure/gpu-tests.yml"
+      - "examples/run_ddp_examples.sh"
+      - "examples/convert_from_pt_to_pl/**"
+      - "examples/run_pl_examples.sh"
+      - "examples/pl_basics/backbone_image_classifier.py"
+      - "examples/pl_basics/autoencoder.py"
+      - "examples/pl_loops/mnist_lite.py"
+      - "examples/pl_fault_tolerant/automatic.py"
+      - "examples/test_pl_examples.py"
+      - "examples/pl_integrations/dali_image_classifier.py"
```

Comment:
```suggestion
      - "examples/pl_*/**"
      - "examples/test_pl_*.py"
```

Reply:
Note all pl examples are checked in our CI, so we don't want to do this